features/admin.applications.v1/pages/applications.tsx

```diff
@@ -390,7 +393,11 @@ const ApplicationsPage: FunctionComponent<ApplicationsPageInterface> = (
      * Navigate to the my account edit page.
      */
     const navigateToMyAccountSettings = (): void => {
-        if (applicationConfig?.advancedConfigurations?.showDefaultMyAccountApplicationEditPage) {
+        if (
+            applicationDisabledFeatures?.includes(
+                ApplicationManagementConstants.FEATURE_DICTIONARY.get("APPLICATION_MYACCOUNT_SAAS_SETTINGS")
```

Comment:
We can use the `isFeatureEnabled` method from core module for this.

Reply:
Will send a follow up PR for this.